source/hardware/connecting-to-the-serial-console.rst

```diff
@@ -93,6 +93,12 @@ port, start PuTTY and then set the following parameters:
 The session may be saved for later use. Click **Open** to start the
 serial console session.
 
+MINICOM
+~~~~~~~
+code-block:: console
```

Comment:
Needs a blank line above here.

And it needs ``.. `` before ``code-block``. It should be: ``.. code-block:: console``